bugbug/repository.py

```diff
@@ -415,6 +505,23 @@ def _transform(commit):
     commit.maximum_test_file_size = max(test_sizes, default=0)
     commit.minimum_test_file_size = min(test_sizes, default=0)
 
+    if metrics_file_count:
+        commit.average_cyclomatic /= metrics_file_count
+        commit.average_halstead_unique_operands /= metrics_file_count
+        commit.average_halstead_operands /= metrics_file_count
+        commit.average_halstead_unique_operators /= metrics_file_count
+        commit.average_halstead_operators /= metrics_file_count
+        commit.average_source_loc /= metrics_file_count
+        commit.average_logical_loc /= metrics_file_count
+    else:
+        commit.minimum_cyclomatic = 0
```

Comment:
Add a comment explaining that we need to do this when we didn't get any metrics and so the minimum would be sys.maxsize